elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,449 @@
+  --_secondary-active-background-color: transparent;
+  --_secondary-active-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-focus-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-background-color: transparent;
+  --_secondary-focus-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_secondary-hover-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-background-color: transparent;
+  --_secondary-hover-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* TERTIARY */
+  --_tertiary-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-background-color: transparent;
+  --_tertiary-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-border-width: var(--rh-border-width-sm, 1px);
+  --_tertiary-active-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-active-background-color: transparent;
+  --_tertiary-active-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-active-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-focus-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-focus-background-color: transparent;
+  --_tertiary-focus-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_tertiary-hover-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-hover-background-color: transparent;
+  --_tertiary-hover-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-hover-border-width: var(--rh-border-width-md, 2px);
+
+  /* LINK */
+  --_link-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_link-background-color: transparent;
+  --_link-active-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-active-background-color: transparent;
+  --_link-focus-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-focus-background-color: transparent;
+  --_link-hover-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-hover-background-color: transparent;
+
+  /* CLOSE */
+  --_close-color: var(--rh-context-light-color-text-muted, #6a6e73);
+  --_close-background-color: transparent;
+  --_close-active-color: var(--rh-color-black-900, #151515);
+  --_close-active-background-color: transparent;
+  --_close-focus-color: var(--rh-color-black-900, #151515);
+  --_close-focus-background-color: transparent;
+  --_close-hover-color: var(--rh-color-black-900, #151515);
+  --_close-hover-background-color: transparent;
+
+  /* PLAY */
+  --_play-color: white;
+  --_play-background-color: #898989; /* WARNING: not a token value */
```

Comment:
`--_play-background-color: var(--rh-color-black-900, #151515);`
`--_play-background-opacity: var(--rh-opacity-50, 50%);`
`--_play-active-background-color: var(--rh-color-black-900, #151515);`
`--_play-active-background-opacity: var(--rh-opacity-80, 80%);`
`--_play-focus-background-color: var(--rh-color-black-900, #151515);`
`--_play-focus-background-opacity: var(--rh-opacity-80, 80%);`
`--_play-hover-background-color: var(--rh-color-black-900, #151515);`
`--_play-hover-background-opacity: var(--rh-opacity-80, 80%);`

Reply:
this should be handled with the new (auto-generated!) `--rh-color-${crayon}-${tone}-rgb` tokens